CHANGELOG.md

```diff
@@ -23,12 +23,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
    the non-coalesced case.
  - decoder API: new function `JxlDecoderGetExtraChannelBlendInfo` to get
    the blending information for extra channels in the non-coalesced case.
+ - encoder API: added ability to set several encoder options to frames using
+   `JxlEncoderSetFrameSetting`
 
 ### Changed
 - decoder API: using `JxlDecoderCloseInput` at the end of all input is required
   when using JXL_DEC_BOX, and is now also encouraged in other cases, but not
   required in those other cases for backwards compatiblity.
 - encoder API: `JxlEncoderCloseInput` now closes both frames and boxes input.
+- encoder API: renamed `JxlEncoderOptions` to `JxlEncoderFrameSettings`, and
+  renamed related functions such as `JxlEncoderOptionsCreate` to
+  `JxlEncoderFrameSettingsCreate`. The old names are kept available but are
+  marked as deprecated.
```

Comment:
It might be worth specifically listing all function/struct names that have become deprecated, and what the new name is. That way an application dev can update their code just by looking at the Changelog, without having to dig in in the .h files or api documentation.

Perhaps instead of putting this under `### Changed`, it makes sense to have another section `### Deprecated` for this.

Reply:
Done